app/views/petition_mailer/privacy_policy_update_email.html.erb

```diff
@@ -0,0 +1,167 @@
+<p>Dear <%= @name %></p>
+
+<p>We're getting in touch to let you know we’re making changes to the privacy notice for the UK Government and Parliament Petitions site (<%= link_to(nil, home_url) %>) on 1st December 2021. We are contacting you to make you aware of these changes because you have started/signed the following petitions that will be affected by these changes:</p>
+
+<% @petitions.each do |petition| %>
+  <p>
+    <strong><%= petition.action %></strong>
+    <br>
+    <%= link_to(nil, petition_url(petition)) %>
+  </p>
+<% end %>
+
+<% if @remaining_petition_count > 0 %>
+  <p>
+    Plus another
+    <%= pluralize(@remaining_petition_count, "petition", locale: :en) %>
+  </p>
+<% end %>
+
+<p>We retain your personal data for as long as is necessary for the purpose it was collected.  We will never share or sell your personal data to other organisations for direct marketing purposes.</p>
+
+<p>What's changing and why?</p>
+
+<ul>
+  <li>The privacy notice has been updated. This is to ensure the privacy notice is up to date and complies with all relevant legislation, including the United Kingdom General Data Protection Regulation (UK GDPR) and Data Protection Act 2018.</li>
+
+  <li>The House of Commons is now the sole data controller for the site. This update reflects the way that your data is stored and used.</li>
+
+  <li>Information about cookies has been removed.  This update is because the site no longer uses cookies to collect information about how you use the site.</li>
+
+  <li>How long we keep your data.  Personal data was kept for 6 months after a petition closed, but this has meant that the Petitions Committee hasn't been able to contact petitioners who opt-in to updates about their petitions when a debate has been scheduled or when other relevant Parliamentary business has taken place. This update means that from 1st November, we will hold your personal data for six months after the dissolution of the current Parliament. Your data protection rights are not affected by this change and you can exercise those rights at any time as detailed in the privacy notice.</li>
+</ul>
+
+<p>You can read the updated privacy notice below or at <%= link_to(nil, privacy_url) %>. This includes information about how to contact us.</p>
+
+<p>
+  <%= t("petitions.emails.signoff_prefix") %>
+  <br>
+  <%= t("petitions.emails.signoff_suffix") %>
+</p>
+
+
+<h1>Privacy notice for the Petitions Committee</h1>
```

Comment:
It might make sense to extract the text of "views/pages/privacy" into a partial and then render it here, rather than include all of the below?

Although not until https://github.com/unboxed/e-petitions/pull/17 is merged, to avoid merge conflicts!

Reply:
Yes, that should be fine